src/components/Board.vue

```diff
@@ -2,31 +2,23 @@
   <div>
     <h1>{{ board.name }}</h1>
     <h2>Label Filter</h2>
-    <label for="labelFilterOption">Label Filter Option: </label>
-    <select id="labelFilterOption" v-model="labelFilterOptionModel">
-      <option value="filter">Filter</option>
-      <option value="select">Select</option>
-    </select>
-    <div v-if="labelFilterOptionModel === 'select'">
-      <select id="labelFilterOption" v-model="labelSelect">
-        <option
-          v-for="labelOption in selectedLabelOptions"
-          v-bind:value="labelOption.value"
-          v-bind:key="labelOption.value">{{labelOption.label}}</option>
-      </select>
+    <div class="label-box-conatiner">
+      <div v-for="labelOption in labelOptions" class="label-box">
```

Comment:
Elements in iteration expect to have 'v-bind:key' directives  vue/require-v-for-key